_maps/shuttles/hunter_russian.dmm

```diff
@@ -136,7 +136,7 @@
 /turf/open/floor/mineral/plastitanium/red,
 /area/shuttle/hunter)
 "A" = (
-/obj/machinery/computer/shuttle/pirate/hunter{
+/obj/machinery/computer/shuttle/hunter{
```

Comment:
Actualy no hold on, this type doesn't even exist?

Reply:
i'll leave a comment where this is defined.